src/components/pill/components.test-pw.tsx

```diff
@@ -1,8 +1,21 @@
 import React from "react";
 import Pill, { PillProps } from ".";
+import Box from "../box";
 
-const PillComponent = ({ children = "noop", ...args }: Partial<PillProps>) => {
+export const PillComponent = ({
+  children = "noop",
+  ...args
+}: Partial<PillProps>) => {
   return <Pill {...args}>{children}</Pill>;
 };
 
-export default PillComponent;
+export const PillOnDarkBb = ({
```

Comment:
nitpick(non-blocking): try and avoid shorthand where possible

It doesn't really matter as it's just a test but I'd personally prefer to try and just be fully descriptive of what it is. I also thing you meant `Bg` as opposed to `Bb` anyway.